__tests__/index.test.js

```diff
@@ -246,20 +210,55 @@ describe(`PATCH /api/articles/:article_id tests`, () => {
         })
     })
     describe(`Error handling tests`, () => {
-        test(`404 - Path not found for /api/topi`, () => {
+        test(`400 - No article with ID 666`, () => {
             return request(app)
-            .get('/api/artivle/1')
+            .get(`/api/articles/666`)
             .expect(404)
             .then(({ body }) => {
-                expect(body.msg).toBe('Route not found');
-            });
+                expect(body.msg).toBe(`No articles with ID: 666`);
+            })
         })
-        test(`400 - No article with ID 666`, () => {
+    })
+})
+
+describe(`POST /api/articles/:article_id/comments tests`, () => {
+    describe(`POST tests`, () => {
+        test(`POST /api/articles/:article_id/comments creates a new comment and returns the posted comment`, () => {
+            const postThis = {
+                username: "icellusedkars",
+                body: "Russia's defence ministry says some troops positioned on the border with Ukraine are returning to their bases after completing drills"
+            };
             return request(app)
-            .get(`/api/articles/666`)
+            .post(`/api/articles/4/comments`)
+            .send(postThis)
+            .expect(201)
+            .then(({ body }) => {
+                const { comment } = body;
+                expect(comment).toEqual(
+                    expect.objectContaining({
+                        comment_id: expect.any(Number),
+                        body: "Russia's defence ministry says some troops positioned on the border with Ukraine are returning to their bases after completing drills",
+                        article_id: 4,
+                        author: "icellusedkars",
+                        votes: 0,
+                        created_at: expect.any(String)
+                    })
+                )
+            })
+        })
+    })
+    describe(`Error handling tests`, () => {
+        test.only(`400 - No article with ID 666`, () => {
+            const postThis = {
+                username: "icellusedkars",
```

Comment:
- TEST - what if there are missing keys, i.e. just the username
- TEST - What is the values are the wrong i.e. numbers instead of string. 
-  TEST - what if the username doesnt exist?
- TEST - What if extra key on the post object?

Reply:
The first 3 have now got tests, the last case isn't valid due to how the values are deconstructed, but I might be wrong.